### ISSUE TITLE
Question: different print setup settings per label types

### ISSUE THREAD

**Issue by kellyaquinn** [ExLibrisGroup/SpineOMatic#25]
Hi, is it possible to have print setup settings for the spine label and different settings for the custom label? I would like the print setup settings to change when the user switches between the spine label and the custom label type. I apologize if this is covered in the documentation. Thanks.

**Comment by mgobat** [comment]
No, the print setup panel settings are not stored separately in the configuration file for "Spine" and "Custom".  Let's flag this one as an enhancement request.

**Comment by panamintsteve** [comment]
> Hi, is it possible to have print setup settings, specifically font size, for the spine label and different settings for the custom label? I would like the print setup settings to change when the user switches between the spine label and the custom label type. I apologize if this is covered in the documentation. Thanks.

Hello Kelly and mgobat,
We have the same interest here.
Thank you.

**Comment by Ruchiya** [comment]
We have the same interest here - would really like to see a way that we can switch around different label setting easily (e.g. able to save particular setting as a template)  without the need to re-enter the field every time

**Comment by vpnakkeow** [comment]
Previously I would run 2 instances of SpineOMatic v.6. Each instance used the settings saved for a specific label type. I can't seem to do that now with v.8.1.2. So, yes, can we have a way to save multiple label settings and then be able to switch between them? Please and thank you.
